src/elements/content-sidebar/activity-feed/task-form/TaskForm.tsx

```diff
@@ -0,0 +1,526 @@
+                            shouldSetActiveItemOnOpen
+                            shouldClearUnmatchedInput
+                            validateForError={() => this.validateForm('taskAssignees')}
+                        >
+                            {approverOptions.map(({ id, name, item = {} }) => (
+                                <ContactDatalistItem
+                                    key={id}
+                                    data-testid="task-assignee-option"
+                                    name={name}
+                                    subtitle={
+                                        (item as UserMini | GroupMini).type === 'group' ? (
+                                            <FormattedMessage {...messages.taskCreateGroupLabel} />
+                                        ) : (
+                                            (item as UserMini).email
+                                        )
+                                    }
+                                />
+                            ))}
+                        </PillSelectorDropdown>
+                        {shouldShowCompletionRule && (
+                            <>
+                                <FeatureFlag feature="activityFeed.tasks.assignToGroup">
+                                    <Checkbox
+                                        data-testid="task-form-completion-rule-checkbox-group"
+                                        isChecked={isCompletionRuleCheckboxChecked}
+                                        isDisabled={isCompletionRuleCheckboxDisabled || isForbiddenErrorOnEdit}
+                                        label={<FormattedMessage {...messages.taskAnyCheckboxLabel} />}
+                                        tooltip={intl.formatMessage(messages.taskAnyInfoGroupTooltip)}
+                                        name="completionRule"
+                                        onChange={this.handleCompletionRuleChange}
+                                    />
+                                </FeatureFlag>
+                                <FeatureFlag not feature="activityFeed.tasks.assignToGroup">
+                                    <Checkbox
+                                        data-testid="task-form-completion-rule-checkbox"
+                                        isChecked={isCompletionRuleCheckboxChecked}
+                                        isDisabled={isCompletionRuleCheckboxDisabled || isForbiddenErrorOnEdit}
+                                        label={<FormattedMessage {...messages.taskAnyCheckboxLabel} />}
+                                        tooltip={intl.formatMessage(messages.taskAnyInfoTooltip)}
+                                        name="completionRule"
+                                        onChange={this.handleCompletionRuleChange}
+                                    />
+                                </FeatureFlag>
+                            </>
+                        )}
+
```

Comment:
Please remove this empty line. Do not include any other change besides remove this line here at line 456.